subprojects/docs/src/snippets/ant/useExternalAntTask/groovy/build.gradle

```
@@ -5,7 +5,7 @@ repositories {
 configurations.create("checkstyle")
 
 dependencies {
-    checkstyle 'com.puppycrawl.tools:checkstyle:8.45.1'
+    checkstyle 'com.puppycrawl.tools:checkstyle:10.4'
```

:x: Let's use 9.3 here so it works for everyone, including folks using Java < 11
```suggestion
    checkstyle 'com.puppycrawl.tools:checkstyle:9.3'
```